wal/wal.go

```diff
@@ -499,6 +499,13 @@ func (w *WAL) Log(recs ...[]byte) error {
 // the final record of a batch, the record is bigger than the page size or
```

Comment:
Can we modify this comment to be bullet list ? Would easier to enumerate those 3 options.

Reply:
updated